testgrid/config/BUILD

```diff
@@ -1,8 +1,9 @@
+load("@io_bazel_rules_go//proto:def.bzl", "go_proto_library")
```

Comment:
We should probably keep using the legacy proto option with gazelle until we address https://github.com/bazelbuild/rules_go/issues/907, since otherwise it does broken things to this BUILD file.